app/models/user.rb

```diff
@@ -274,4 +274,8 @@ def send_devise_notification(notification, *args)
 
   def may_create_profile? = reputation >= User::Profile::MIN_REPUTATION
   def profile? = profile.present?
+
+  def confirmed? = super && !disabled? && !blocked?
```

Comment:
I've ordered the checks to have the non-expensive checks first

Reply:
Nice.